### PR TITLE
a bit cleanup tile models

### PR DESCRIPTION
**Pull request by benrudolph**
ran into this, couldn't get case tiles to work without formplayer crashing, do you mind giving this a spin?

@wpride

### CODE REVIEW

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/menus/menu_view.js [hunk 1]
```diff
@@ -100,30 +100,26 @@ FormplayerFrontend.module("SessionNavigate.MenuList", function (MenuList, Formpl
...
+            .map(function(tile, idx) {
+                return {
+                    id: 'grid-style-' + idx,
+                    gridStyle: getGridAttributes(tile),
+                    fontStyle: tile.fontSize,
+                };
+            }).value();
+
+        templateString = $("#case-tile-style-template").html();
+        tileStyleTemplate = _.template(templateString);
```
Comment (dimagimon) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/menus/menu_view.js]: 'tileStyleTemplate' is not defined. (no-undef)

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/menus/menu_view.js [hunk 2]
```diff
@@ -100,30 +100,26 @@ FormplayerFrontend.module("SessionNavigate.MenuList", function (MenuList, Formpl
...
+        tileStyle = tileStyleTemplate({
```
Comment (dimagimon) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/menus/menu_view.js]: 'tileStyleTemplate' is not defined. (no-undef)

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/menus/menu_view.js [hunk 3]
```diff
@@ -100,30 +100,26 @@ FormplayerFrontend.module("SessionNavigate.MenuList", function (MenuList, Formpl
...
-                if (tile === null) {
-                    continue;
-                }
-                fontSize = tiles[i].fontSize;
-                fontString = fontSize;
-                styleString = getGridAttributes(tile);
-                tileId = "grid-style-" + i;
-                obj.id = tileId;
-                obj.gridStyle = styleString;
-                obj.fontStyle = fontString;
```
Comment (benrudolph) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/menus/menu_view.js]: `tiles[i].fontSize -> fontSize -> fontString -> obj.fontStyle` haha

Comment (wpride) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/menus/menu_view.js]: :classic: :shame:

**Comment by wpride** [comment]
Confirmed this still works once merged with the fix in https://github.com/dimagi/commcare-hq/pull/12772, merging

**Pull request merged**